DESCRIPTION

```diff
@@ -27,7 +27,8 @@ Imports:
     tibble (>= 1.4.2),
     tools,
     withr (>= 1.0.0),
-    xfun (>= 0.1)
+    xfun (>= 0.1),
+    R.cache (>= 0.13.0.9000)
```

Comment:
Did you mean to add {R.cache} to "Suggests"? Leaving it in "Imports" seems easier.

Reply:
I see it was only added in the very last commit `cb2dc17363dfaa39f4025a1895ffed86556e5ceb` when I added my own fork as a remote dependency. Probably caused by unintentional `usethis::use_dep()`. Do you think it's unnecessary to keep it in `Suggests`? It needs some extra handling in other places, I agree. 
``` r
setdiff(miniCRAN::pkgDep("R.cache"), miniCRAN::pkgDep("styler"))
#> [1] "R.cache"     "R.methodsS3" "R.oo"        "R.utils"
```

<sup>Created on 2019-09-23 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>